CONTRIBUTING.md

```diff
@@ -37,6 +37,21 @@ or
 $ npm run make:login
 ```
 
+## Deploy RhodsApplications, RhodsDocuments to the OpenShift cluster
+   1. Make sure your oc project is set
+      ```
+      oc project $OC_PROJECT
+      ```
+
+   2. Deploy the RhodsApplications, RhodsDocuments CRDS
+      ```
+      oc apply install/deploy/base/custom-resource-definition.yaml
+      ```
+
+   3. Deploy the RhodsApplications, RhodsDocuments Resources to your OC_PROJECT
```

Comment:
I think we need to set the project correctly first: `oc project <rhods project>`

Reply:
done